src/status_im/navigation/effects.cljs

```diff
@@ -164,9 +164,9 @@
                                                                                  theme)})
                                                options
                                                (when sheet?
-                                                 options/sheet-options))}}]}})
-        (state/navigation-state-push {:id   component
-                                      :type :modal})))))
+                                                 options/sheet-options))}}]}})))
+    (state/navigation-state-push {:id   component
+                                  :type :modal})))
```

Comment:
revert from https://github.com/status-im/status-mobile/pull/22008#discussion_r1939026085

Intended change in that PR: only push the modal to the state when opening it

Side Effect: When we quickly close the old modal and open a new one, we store the new modal in state/dismissing (Line 155) and open it after the old modal has been dismissed. However, when the new modal is opened, the view-id has not been updated yet, which leads to incorrect values for subscriptions such as :get-screen-params. Therefore, we are reverting this change to update the view-id by pushing the modal in state as soon as it is opened.